code/espurna/ws.ino

```diff
@@ -139,7 +139,7 @@ void _wsParse(AsyncWebSocketClient *client, uint8_t * payload, size_t length) {
 
     // Check actions -----------------------------------------------------------
 
-    const char* action = root["action"];
+    const char* action = root["data"]["command"];
```

Comment:
As far as I can see we are still sending 
```
function sendAction(action, data) {
    websock.send(JSON.stringify({action: action, data: data}));
}
```
actions: .... in our custom.js so what this line actually fixes ?